backend/src/functions/onCardDrawn.ts

```diff
@@ -29,23 +31,23 @@ export const handler = async (event: APIGatewayEvent) => {
     const currentPlayerIndex = playerIds.indexOf(game.currentPlayerId)
     const nextPlayerId = game.players[(currentPlayerIndex + 1) % game.players.length].id
 
-    const updatedGame: IGameModel = {
+    updatedGame = {
       ...game,
       currentPlayerId: nextPlayerId,
       players: updatedGamePlayers,
     }
     console.log("updating game in db")
     await database.room.updateGame({ roomId, game: updatedGame })
```

Comment:
super nit and unrelated to your work but we can move line 39/40 and 46/47 out of the if block now

Reply:
good point!